R/pkg/R/SQLContext.R

```diff
@@ -328,6 +328,7 @@ setMethod("toDF", signature(x = "RDD"),
 #' It goes through the entire dataset once to determine the schema.
 #'
 #' @param path Path of file to read. A vector of multiple paths is allowed.
+#' @param ... additional external data source specific named properties.
```

Comment:
Hm, do you think it is okay as it is? I tried to make them look more consistent but it kind of failed.